cg/apps/gt.py

```diff
@@ -21,6 +25,10 @@ def __init__(self, config: dict):
         alchy_config = dict(SQLALCHEMY_DATABASE_URI=config['genotype']['database'])
         super(GenotypeAPI, self).__init__(config=alchy_config, Model=models.Model)
 
+        self.genotype_database = config['genotype']['database']
```

Comment:
could we use the config rather the specifics in the config

Reply:
Yes, that's definitely the way to go. Loqusdb does this already so you'll find an example on how to do this for cg and for the config

Reply:
I've checked now and there is a config for genotype, please consider using it like Adam did in Loqus